src/styles/renderer/_catdistributionbar.scss

```diff
@@ -10,5 +10,17 @@
     display: flex;
     flex-direction: row;
     align-items: center;
+    position: relative;
+    color: transparent;
+
+    &:before {
+      content: attr(data-cat);
+      position: absolute;
+      left: 5px;
+      right: 3px;
+      overflow: hidden;
+      text-overflow: ellipsis;
+      color: var(--text-color, black); // custom property `--text-color` is set in CategoricalStackedDistributionlRenderer.ts
```

Comment:
does every browser (firefox esr, chrome, edge) support that?

Reply:
Yes, every new browser: https://caniuse.com/#feat=css-variables

Reply:
what happens it is isn't supported, such as in in Firefox ESR 52...

Reply:
The solution is to use `@supports` as you can see here: https://github.com/Caleydo/lineupjs/pull/506/commits/54811aa1aeb436656cdc404edf201724e00cda72